php-config/Slate/Connectors/GSuite/API.config.d/google-credentials.php

```diff
@@ -0,0 +1,20 @@
+// API::$domain = 'scienceleadership.org';
+
+API::$clientId = '116995110434775517191';
+API::$clientEmail = 'calendar@calendar-project-272323.iam.gserviceaccount.com';
+API::$privateKey = "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCjCuugaq6Z372n\n3DKogCVmUZnVz/35mQ8x6W1LBfkN77R5OlwWjodvkjGeRVzYWvy1G+wcl6Y0GCcf\nzAPdH4hrrgzpMhmUPlGTq8cKk9htTyaV+JDyb7sXLIRvBPt4h8rpuzqSLn5+wGAq\no/RRoBTWRAnfU833sn1/SVzz5uNz1uGQE0mog+LlL0rJdWhI0N9edxvO4pR2dNGY\n/T6TBi3N5zkuUhH9wCmrnz15q2nBg6wmsvyN5r3NBXRVBbMGp0UJQeK20nsN2rLV\nRTo7Zhry/h2aCaKvy9mPj1BLlegaODWY/QpHxYI+WzCOX7amCYIbCa2pmZmzMZVS\nD5JMCq81AgMBAAECggEAAMEk/+Guq7IGFsearbKytiqV8UnjABn5ejg7IBTC6Jod\n/JU8+41lHy1fbgJjNummFDbWHHEYEZq4QWRWW5g9VJiY8k9vtPkn8uyfft50UXYe\n13YA2dYXg6E6nNyw0zvwYmtMtrAzks++PxVMsVXljRw8Ad6T3FhpNaKGpo7CZ2EC\nvaiwaaV/Yydc/YzBxZN9HgvYc+GZtS8G+5K+3u6Zfd90N/XmR0NK2I/pZFNpFA2+\nJzqd3KK4yYCPvMIrMuJQFDEaQwex0NkI28KsD+ook6UJQ3ZQBMF+SM4WioN5C1qE\nmZBPSltFe5mUfbsyGPtUPBXVNG90zIvtXuEzWTeq0QKBgQDhnsWzRwKlu3OXw/XW\ntEHc+3i0/RRUdLNkPMAbXEo6iso1xvKqVYvg/wR4EpXnsZPwiPqVboN7UTOnnXX/\neRI5sr4/p+EiqIJHtJHGiu6odNG22gIup6JBZVlH4FDpFy8gWSI+T8nI2jn0xG28\nHinTRLwnSRauL7Lcu5VHsM38ewKBgQC4/xKVK/kyFYxg7Fno/53WETdvp6m6MHq0\nwl0OxdlmiYNNrHStw8dTZg5bS4DYFwf5f3Gp8Pd/eMHCMweWKdeGvJeaodKkSp0W\nOX7ZBEa+hbUFIsdWgIOu6dnt/PByjQIUKfTML9bTpliPnzjiouFFK4yEf6Y4oItx\noneKBs9sDwKBgGzc9TupucdKCV5jOU2dsGH2HKoHqzWNEbn97hvFjjNtIGCOXCyA\nFnw70GY3YYloD5/yGCbeKiboaWoo+fSI8JhSfKWuVV0pvaLnp6cLBljV9RKrYn4s\neIfhZRDfVWsCusR6/aDfUtfNJEXMOb6ZN0kvUzIJfB4T+dOz1zW+Q8XdAoGAG8jb\njqH7HGWWHtyAqHjzyLnvPIrO+BLd6xYiCdz3SOowWVMkdxCneEQktduCL3JOFU/p\nz19MnM2TNEqbwhbosujSUDwrudcm/raF8awtv+GvDWVq0PgeTv/6Zj8OEfZRsJuY\nk64/sqwSMwfz8D4MXWosAZ52ZZzfNKL1FzrsnksCgYEAwtQB5tdei9NcftpzEMHx\nmjYQYXxq04zn/ACFuHTjqegWeceKym0naAXBa10nC4AuVH1yUrVcbHku4PWbG8xp\nNdxeU6oCbQnbcBunMzwTf4LILp7R9sX+IoXoIJU8zT5hwwOFw2VWwgK/lg7d5Oxb\nq0OpfzfIp4J+ML66eLrSi10=\n-----END PRIVATE KEY-----\n";
+API::$domain = 'nafis.me';
```

Comment:
let's blank all these out, maybe leave commented-out lines with example placeholders that help people see what the values should look like

Reply:
this wasn't supposed to be committed :/

Reply:
Ok, these credentials have been removed from their respective service accounts.